app/scripts/controllers/app-state-controller.ts

```diff
@@ -196,79 +202,239 @@ const getDefaultAppStateControllerState = (
   currentExtensionPopupId: 0,
 });
 
-export class AppStateController extends EventEmitter {
-  private readonly extension: AppStateControllerOptions['extension'];
+const controllerMetadata = {
+  timeoutMinutes: {
+    persist: true,
+    anonymous: true,
+  },
+  connectedStatusPopoverHasBeenShown: {
+    persist: true,
+    anonymous: true,
+  },
+  defaultHomeActiveTabName: {
+    persist: true,
+    anonymous: true,
+  },
+  browserEnvironment: {
+    persist: true,
+    anonymous: true,
+  },
+  popupGasPollTokens: {
+    persist: true,
+    anonymous: true,
+  },
+  notificationGasPollTokens: {
+    persist: true,
+    anonymous: true,
+  },
+  fullScreenGasPollTokens: {
+    persist: true,
```

Comment:
These tokens are for keeping track of ongoing polling, so I don't think it would be useful to persist them. It's not possible for polling to be active between sessions.

```suggestion
  popupGasPollTokens: {
    persist: false,
    anonymous: true,
  },
  notificationGasPollTokens: {
    persist: false,
    anonymous: true,
  },
  fullScreenGasPollTokens: {
    persist: false,
```